test/binary-op.civet

```diff
@@ -146,3 +146,35 @@ describe "binary operations", ->
     ---
     a instanceof b
   """
+
+  testCase """
+    pipe expression
+    ---
+    a |> fn
+    ---
+    (fn)(a)
```

Comment:
We could check for common cases where the fn is an identifier or already a parenthesized expression so we won't wrap in excess parens.

Ideally this would generate `fn(a)`.